packages/kit/scripts/extract-types.js

```diff
@@ -29,7 +29,7 @@ for (const statement of node.statements) {
 		}
 
 		const i = code.indexOf('export', start);
-		start = i + 6;
+		start = i >= 0 ? i + 6 : start;
```

Comment:
this is wrong, it just gets the index of the next `export` in the file, which means that start > end (hence non-exported types not showing up on https://kit-svelte-dev-git-public-types-svelte.vercel.app/docs/types)

Reply:
reverted